plugins/modules/ipaservice.py

```diff
@@ -839,7 +839,12 @@ def main():
                         changed = True
                 else:
                     changed = True
+            except ipalib.errors.EmptyModlist:
```

Comment:
Why have you added these lines?

Reply:
This change is in a TEMP commit, so service tests pass.